.travis.yml

```diff
@@ -20,19 +21,29 @@ before_install:
     - pecl channel-update pecl.php.net
     - pecl config-set preferred_state beta
     - echo yes | pecl upgrade apcu
+    - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
     - phpenv config-add ./tests/travis/php.ini
-    - if [[ $TRAVIS_PHP_VERSION = '7.1' ]]; then PHPUNIT_FLAGS="--coverage-clover clover.xml"; fi
 
 install:
     - travis_retry composer install
 
-script:
-    - ./vendor/bin/phpunit --exclude-group performance $PHPUNIT_FLAGS
-
-after_script:
-    - if [[ -f clover.xml ]]; then wget https://scrutinizer-ci.com/ocular.phar && php ocular.phar code-coverage:upload --format=php-clover clover.xml; fi
-
-matrix:
-    fast_finish: true
-    allow_failures:
-        - php: nightly
+script: ./vendor/bin/phpunit --exclude-group performance
+
+jobs:
+  allow_failures:
+    - php: nightly
+
+  include:
+    - stage: Coding standard
+      script:
+        - ./vendor/bin/phpcs
+
+    - stage: Coverage
+      before_script:
+        - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{.disabled,}
+        - if [[ ! $(php -m | grep -si xdebug) ]]; then echo "xdebug required for coverage"; exit 1; fi
+      script:
+        - ./vendor/bin/phpunit --exclude-group performance --coverage-clover clover.xml
```

Comment:
Since you rely on phpcs, phpcs should also be a dependency (besides `doctrine/coding-standard`)

Reply:
Alright, will add it then.